Sources/Testing/SourceAttribution/Expression.swift

```diff
@@ -237,58 +258,129 @@ public struct Expression: Sendable {
         value: value.capturingRuntimeValues(firstValue),
         keyPath: keyPath.capturingRuntimeValues(additionalValuesArray.first ?? nil)
       )
+    case let .negation(expression, isParenthetical):
+      result.kind = .negation(
+        expression.capturingRuntimeValues(firstValue, repeat each additionalValues),
+        isParenthetical: isParenthetical
+      )
     }
 
     return result
   }
 
+
+  /// Get an expanded description of this instance that contains the source
+  /// code and runtime value (or values) it represents.
+  ///
+  /// - Returns: A string describing this instance.
+  @_spi(ForToolsIntegrationOnly)
+  public func expandedDescription() -> String {
+    _expandedDescription(in: _ExpandedDescriptionContext())
+  }
+
+  /// Get an expanded description of this instance that contains the source
+  /// code and runtime value (or values) it represents.
+  ///
+  /// - Returns: A string describing this instance.
+  ///
+  /// This function produces a more detailed description than
+  /// ``expandedDescription()``, similar to how `String(reflecting:)` produces
+  /// a more detailed description than `String(describing:)`.
+  func expandedDebugDescription() -> String {
+    var context = _ExpandedDescriptionContext()
+    context.includeTypeNames = true
+    context.includeParenthesesIfNeeded = false
+    return _expandedDescription(in: context)
+  }
+
+  /// A structure describing the state tracked while calling
+  /// `_expandedDescription(in:)`.
+  private struct _ExpandedDescriptionContext {
```

Comment:
`Sendable`?

Reply:
No need to as it never gets sent anywhere.